cmds/report.c

```diff
@@ -240,12 +240,8 @@ static void report_functions(struct uftrace_data *handle, struct opts *opts)
 {
 	struct rb_root name_root = RB_ROOT;
 	struct rb_root sort_root = RB_ROOT;
-	const char f_format[] = "  %10.10s  %10.10s  %10.10s  %-.*s\n";
+	const char f_format[] = "  %10.10s  %10.10s  %10.10s  %-.*s";
 	const char line[] = "=================================================";
-	char field_func_name[] = "Function [Source]";
-
-	if (!opts->srcline)
-		field_func_name[8] = '\0';
```

Comment:
Please move this part into the previous commit so that we can skip unnecessary changes in the first place.